src/Prettus/Repository/Criteria/RequestCriteria.php

```diff
@@ -46,7 +46,7 @@ public function apply($model, RepositoryInterface $repository)
 
         if ($search && is_array($fieldsSearchable) && count($fieldsSearchable)) {
 
-            $searchFields = is_array($searchFields) || is_null($searchFields) ? $searchFields : explode(';', $searchFields);
+            $searchFields = is_array($searchFields) || is_null($searchFields) ? $searchFields : array_filter(explode(';', $searchFields));
```

Comment:
Why not. Think somethink like //domain?false&another_params or //domain?0=value&another_params is crazy.